src/configs/hostname.rs

```diff
@@ -18,6 +20,8 @@ impl<'a> Default for HostnameConfig<'a> {
             ssh_only: true,
             trim_at: ".",
             format: "[$hostname]($style) in ",
+            ssh_format: "[$hostname]($ssh_style) in ",
+            ssh_style: "yellow bold",
```

Comment:
@matchai Thoughts on whether this would be considered breaking?

Right now, the defaults is to not display on localhost and to display in bold dimmed green when SSH'ed.

With current proposed changes, we will not display on localhost and display in bold yellow when SSH'ed.

Possible chance would be to switch ssh_style and style.